diagnostics/diag.go

```diff
@@ -179,7 +179,7 @@ func decodeDiagJson(data []byte) (*DiagInfo, error) {
 func (d *Diagnostics) getDiagnosticFromPeers(ctx context.Context, peers map[peer.ID]int, pmes *pb.Message) (<-chan *DiagInfo, error) {
 	respdata := make(chan *DiagInfo)
 	wg := sync.WaitGroup{}
-	for p, _ := range peers {
+	for p := range peers {
```

Comment:
Btw ```gofmt -s``` compatibility for 1.3 was fixed here https://go-review.googlesource.com/#/c/2562/. Also I was wondering why https://github.com/ipfs/go-ipfs#install says 1.4+ while travis build has 1.3.